include/zephyr/net/wifi.h

```diff
@@ -11,8 +11,10 @@
  */
 
 /**
- * @defgroup wifi_mgmt Wi-Fi Management
  * @brief Wi-Fi Management API.
+ * @defgroup wifi_mgmt Wi-Fi Management
+ * @since 1.12
+ * @version 0.8.0
```

Comment:
missing `wifi_mgmt.h`?

Reply:
`wifi_mgmt.h` does not define a separate group, it uses `@addtogroup wifi_mgmt` so it's merged with `wifi_mgmt` doxygen group defined here.